OpenRA.Mods.Common/Traits/Player/MissionObjectives.cs

```diff
@@ -168,7 +168,7 @@ void CheckIfGameIsOver(Player player)
 				});
 		}
 
-		public void OnPlayerWon(Player player)
+		void INotifyObjectivesUpdated.OnPlayerWon(Player player)
```

Comment:
Why can this trait stay not-conditional?

Reply:
I moved the two traitinfo settings to the GameMode trait, so this can now *really* stay unconditional.